python/dgl/__init__.py

```diff
@@ -50,6 +50,7 @@
 from . import optim
 from .frame import LazyFeature
 from .utils import apply_each
+from .sparse import is_libxsmm_enabled, use_libxsmm
```

Comment:
It is strange to have these APIs in `sparse.py`. How about adding a `global.py` for all global configuration APIs?

Reply:
Make sense, as originally I just focused on spmm usage, let me add a "global.py" for future extension.